app/Http/Controllers/TaggingController.php

```diff
@@ -9,12 +9,12 @@
 
 class TaggingController extends Controller
 {
-    public function showGames($tagid){
+    public function showGames($tagid) {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->